packages/@aws-cdk/aws-ec2/lib/vpc-lookup.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Properties for looking up an existing VPC.
+ *
+ * The combination of properties must specify filter down to exactly one
+ * non-default VPC, otherwise an error is raised.
+ */
+export interface VpcLookupOptions {
```

Comment:
Why does this get to be in it's own file? I'm seeing lots of these crop up and seems to be quite a departure from the previous/current code organization.

Reply:
@rix0rrr this modules needs file reorg anyway. The vpc.ts file is huge and should probably be split to Vpc and Subnets anyway... Let's leave this for later.